.github/workflows/main.yml

```diff
@@ -25,13 +27,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          submodules: recursive
 
       - name: Use Bun 1.2.0
         uses: oven-sh/setup-bun@v2
         with:
           bun-version: 1.2.0
 
-      - run: bun install --frozen-lockfile
+      - run: bun install
```

Comment:
frozen lockfiles ensure no random transititves get pulled and that the build is reproducible. I suggest to put it back and regenerate the lockfile and commit it to ensure it matches the same one in CI